src/Model/GroundWaterEnergy/gwe-est.f90

```diff
@@ -636,9 +766,22 @@ subroutine read_options(this)
         case ('FIRST_ORDER_DECAY')
           this%idcy = 1
           write (this%iout, fmtidcy1)
-        case ('ZERO_ORDER_DECAY')
+        case ('ZERO_ORDER_DECAY_WATER')
```

Comment:
Should this be ZERO_ORDER_DECAY_AQUEOUS to be consistent elsewhere?

Reply:
Based on the [follow-up](https://github.com/MODFLOW-USGS/modflow6/pull/2155#issuecomment-2627459652) comment below, I'm going to leave this as-is for the time being.